meta-mender-toradex-nxp/classes/image_type_mender_tezi.bbclass

```diff
@@ -0,0 +1,195 @@
+FULL_IMAGE_SUFFIX_mender-image-gpt = "gptimg"
+
+TEZI_AUTO_INSTALL ??= "false"
+TEZI_CONFIG_FORMAT ??= "2"
+
+# Do not include these image types:
+IMAGE_FSTYPES:remove = "${SOC_DEFAULT_IMAGE_FSTYPES} tar.xz ${FULL_IMAGE_SUFFIX}.bz2"
+
+WKS_FILE_DEPENDS:append = " mender-tezi-metadata "
+
+do_image_mender_tezi[recrdeptask] += "do_deploy"
+RM_WORK_EXCLUDE += "${PN}"
+
+def rootfs_mender_tezi_emmc(d):
+    from collections import OrderedDict
+    offset_bootrom = d.getVar('OFFSET_BOOTROM_PAYLOAD')
+    offset_spl = d.getVar('OFFSET_SPL_PAYLOAD')
+    imagename = d.getVar('IMAGE_LINK_NAME')
+    image_suffix = d.getVar('FULL_IMAGE_SUFFIX')
+    storage_device = os.path.basename(d.getVar('MENDER_STORAGE_DEVICE'))
+
+    bootpart_rawfiles = []
+
+    if offset_spl:
+        bootpart_rawfiles.append(
+              {
+                "filename": d.getVar('SPL_BINARY'),
+                "dd_options": "seek=" + offset_bootrom
+              })
+    bootpart_rawfiles.append(
+              {
+                "filename": d.getVar('UBOOT_BINARY_TEZI_EMMC'),
+                "dd_options": "seek=" + (offset_spl if offset_spl else offset_bootrom)
+              })
+
+    return [
+        OrderedDict({
+          "name": storage_device,
+          "table_type": "gpt",
+              "content": {
+                  "rawfiles": [
+                      {
+                          "dd_options": "bs=8M",
+<<<<<<< HEAD
+<<<<<<< HEAD
+                          "filename": "%s.%s.gz" % (imagename, image_suffix)
+=======
+                          "filename": "%s.%s" % (imagename, image_suffix)
+>>>>>>> mender-tezi: Remove bz2 compression.
+=======
+                          "filename": "%s.%s.gz" % (imagename, image_suffix)
+>>>>>>> mender-tezi: Add gz compression.
+                      }
+                  ]
+              }
+        }),
+        OrderedDict({
+          "name": "%sboot0" % (storage_device),
+          "erase": True,
+          "content": {
+            "filesystem_type": "raw",
+            "rawfiles": bootpart_rawfiles
+          }
+        })]
+
+def rootfs_mender_tezi_rawnand(d):
+    from collections import OrderedDict
+
+    return [
+        OrderedDict({
+          "name": "u-boot1",
+          "content": {
+            "rawfile": {
+              "filename": d.getVar("UBOOT_BINARY"),
+              "size": 1
+            }
+          }
+        }),
+        OrderedDict({
+          "name": "u-boot2",
+          "content": {
+            "rawfile": {
+              "filename": d.getVar("UBOOT_BINARY"),
+              "size": 1
+            }
+          }
+        }),
+        OrderedDict({
+          "name": "u-boot-env",
+          "content": {
+            "rawfile": {
+              "filename": "uboot.env",
+              "size": 1
+            }
+          }
+        }),
+        OrderedDict({
+          "name": "ubi",
+          "content": {
+            "rawfile": {
+              "dd_options": "bs=8M",
+<<<<<<< HEAD
```

Comment:
Another broken merge?

Reply:
Same as above.  Reverted, then fixed in the later commit.